types/simple-get/index.d.ts

```diff
@@ -0,0 +1,57 @@
+// Type definitions for simple-get 4.0
+// Project: https://github.com/feross/simple-get
+// Definitions by: Konrad Perlicki <https://github.com/KonradPerlicki>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node" />
+
+import { ClientRequest, IncomingMessage, RequestOptions } from 'http';
+import { URL } from 'url';
+import * as FormData from 'form-data';
```

Comment:
The JS module does not have this external dependency:
https://github.com/feross/simple-get/blob/master/package.json
is this one really required? Adding this could impact people existing setup
Can this be somehow patched, if really needed?

Reply:
So in this case, when it is actually allowed to use FormData as body value, is it better to ignore that one type or replace it with `unknown`? I'm not sure which is better solution

Reply:
I suspect that this should just be `stream.Readable`, which both `fs.ReadStream` and `FormData` extend. The underlying implementation just checks for a read method.

Reply:
You're right, thanks for your comment